### ISSUE TITLE
What responsive breakpoint should I use?

### ISSUE THREAD

**Issue by tmurren** [ktownecreative/Coding-Challenge#4]
It seems that there are mobile and desktop designs for this challenge, and I understand that their content areas have max widths of 563px and 960px (as described in point #7), but what would you like the actual breakpoint to be? Can would assume 1508px based on the width of the mobile design?

"7. Have the main content centered and a max width of 563px in mobile view and 960px in desktop."

**Comment by dannyk3941** [comment]
No, the 1508px breakpoint is way past the desktop breakpoint so it would not make sense for the width of the mobile design to be there. 

The mobile design should switch to the desktop design between a screen width of 564px and 960px. You don't have to set any other breakpoints beyond the 960px screen width.
